### ISSUE TITLE
/etc/snapserver.conf not found on armhf

### ISSUE THREAD

**Issue by lyghtnox** [djmaze/librespot-snapserver#9]
Hi,
When I try to create a container with your image I get an error:
```
[alarm@rpi4 ~]$ sudo docker run --name snapserver --net host -e DEVICE_NAME=Snapcast mazzolino/librespot-snapserver
sed: can't read /etc/snapserver.conf: No such file or directory
```
Is the image broken?

**Comment by plains203** [comment]
Same problem here

**Comment by JustMaier** [comment]
Glad to hear I wasn't the only one experiencing this.
I guess 😂

It seems like it's something that's only a problem in the latest release. If I pull the `0.19.0` image it seems to start up.

**Comment by djmaze** [comment]
Yes, I tried to workaround the current problems with building for armv7 and messed the latest build up on this occasion.. Please use `0.19.0` for now (which is the latest version which was released). Sorry for the mess.

**Comment by bbgehring** [comment]
I'm on a rPi 4 running latest version of RaspOS and tried to pull the old 19.0 image for arm devices. When i try to run it snapserver starts ok but librespot fails to start "version GLIBC_2.28 not found (required by /home/localuser/librespot)"

Looks like version 2.27 is running in the container but 2.28 is installed on my host. Any way around this?

**Comment by djmaze** [comment]
@bbgehring Not sure what you are doing there. The image contains both snapserver as well as librespot so you don't need a librespot installation on your host. Just start the container and snapserver will take care of starting librespot as well.

**Comment by bbgehring** [comment]
@djmaze sorry that wasn't the exact error output I was typing from memory. Librespot isn't installed on host. Pulled the previous version of your image for arm and when docker running this is the log I'm receiving from within the container. I just checked to verify the version of GLIBC in the container and it's 2.27. 

> (LibrespotStream) (Spotify) /usr/local/bin/librespot: /lib/arm-linux-gnueabihf/libc.so.6: version `GLIBC_2.28' not found (required by /usr/local/bin/librespot)

**Comment by djmaze** [comment]
Okay, so that's the armv7 version. I just try it out myself. You are right, that version seems broken unfortunately. 

For now, you could go back one more version and use the `0.17.1` tag. That seems to work.

**Comment by Jakobovski** [comment]
Also happens on x86

**Comment by djmaze** [comment]
Please try the latest image, [it should now work on all architectures](https://github.com/djmaze/librespot-snapserver/issues/8#issuecomment-890809386).
